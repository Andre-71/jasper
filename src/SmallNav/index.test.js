import React from "react"
import renderer from 'react-test-renderer'

import SmallNav from '.'

it('renders navItems correctly', () => {
  const navItems = [
    {
      title: "navitem1",
      component: <div>Hello world, this is component1</div>
    },
    {
      title: "navitem2",
      component: <div>Hello world, this is component2</div>
    }
  ]

  const tree = renderer
    .create(<SmallNav navItems={navItems} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
