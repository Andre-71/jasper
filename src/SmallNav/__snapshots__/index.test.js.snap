// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navItems correctly 1`] = `
<div
  className="sc-bdVaJa knAKLA"
>
  <div
    className="small-nav"
  >
    <div
      onClick={[Function]}
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      navitem1
    </div>
    <div
      onClick={[Function]}
      style={
        Object {
          "fontWeight": "normal",
        }
      }
    >
      navitem2
    </div>
  </div>
  <div
    className="nav-content"
  >
    <div>
      Hello world, this is component1
    </div>
  </div>
</div>
`;
